InitGui.py

```diff
@@ -58,8 +58,9 @@ def Initialize(self):
         import SheetMetalRelief
         import SheetMetalJunction
         import SheetMetalBend
+        SketchOnSheetMetalCmd

```

Comment:
doesn't this need to be `import SketchOnSheetMetalCmd`  
if so then I've submitted a PR https://github.com/jaisekjames/FreeCAD_SheetMetal/pull/1

Reply:
Thank you.

Reply:
np, but it still doesn't fix the issue :disappointed:

Reply:
I just downloaded my "new feature" branch zip & pasted on mod folder. It is working. No error.

Reply:
my bad. I wasn't applying the patch correctly. I needed to use `git apply` (i've amended the steps above (https://github.com/shaise/FreeCAD_SheetMetal/pull/146#issuecomment-735216604) to reflect that